wsapi.go

```diff
@@ -951,3 +938,36 @@ func (s *Session) CloseWithCode(closeCode int) (err error) {
 
 	return
 }
+
+func (s *Session) doCloseHandshake(closeCode int) error {
```

Comment:
```suggestion
func (s *Session) gracefulClose(closeCode int) error {
```

Reply:
This function does not actually close the connection though. The caller closes it. All this function does is the handshake.

Reply:
My bad. Not really sure how I missed that 😅